ci/pipeline.yml

```diff
@@ -20,28 +20,29 @@ jobs:
         - get: davcli-src
           resource: davcli-in
           trigger: true
-          passed: [
-            test,
-          ]
-        - {trigger: false, get: version-semver, params: {bump: patch} }
+          passed:
+            - test
 
-      - {put: version-semver, params: {file: version-semver/number}}
+        - put: version-semver
+          params:
+            bump: patch
 
-      - task: build-linux
-        file: davcli-src/ci/tasks/build.yml
-        params:
-          GOOS: linux
-
-      - put: release-bucket-linux
-        params: {file: out/davcli-*-linux-amd64}
-
-      - task: build-windows
-        file: davcli-src/ci/tasks/build.yml
-        params:
-          GOOS: windows
+      - aggregate:
+        - do:
+          - task: build-linux
+            file: davcli-src/ci/tasks/build.yml
+            params:
+              GOOS: linux
+          - put: release-bucket-linux
+            params: {file: out/davcli-*-linux-amd64}
```

Comment:
Oops, seems like we might still end up publishing a linux without a windows if windows fails.

What about...

```
- aggregate:
  - task: build-linux
    file: davcli-src/ci/tasks/build.yml
    params:
      GOOS: linux
  - task: build-windows
    file: davcli-src/ci/tasks/build.yml
    params:
      GOOS: windows
- aggregate:
  - put: release-bucket-linux
    params: {file: out/davcli-*-linux-amd64}
  - put: release-bucket-windows
    params: {file: out/davcli-*-windows-amd64}
```